browser/BUILD.gn

```diff
@@ -153,7 +153,10 @@ source_set("browser_process") {
   }
 
   if (enable_tor) {
-    deps += [ "tor" ]
+    deps += [
```

Comment:
I'm not sure about this one, need to do some research and I'll check back

Reply:
I think we just need "//brave/common/tor:tor_mojom_bindings", here? Checking

Reply:
f6fb36a